### DISCUSSION TITLE
Awareness of Open UI initiative

### DISCUSSION THREAD

**Discussion by bahrus** [Devographics/surveys#191] [State of HTML Design]
https://open-ui.org/

I know the survey's include some sections on sources from which people learn, etc.  This might fall under some category like "Organizations I follow" or something like that.

**Comment by LeaVerou** [comment]
Hey, I moved the discussion since it's not a Feature and edited it to remove all the Feature template stuff.
Good idea! I'll think about how we could potentially make it more general than just Open UI.
